docs/development/how-to/ide-setup/eclipse.md

```diff
@@ -23,7 +23,7 @@
 
 ## Formatter
 1. [Install Google Java Format Plugin](https://github.com/google/google-java-format#eclipse)
-   * ***Important*** Use the unofficial 1.7 version available here: https://github.com/google/google-java-format/files/2774507/google-java-format-eclipse-plugin-1.7.0.jar.zip
+   *[OLD ***Important*** Use the unofficial 1.7 version available here: https://github.com/google/google-java-format/files/2774507/google-java-format-eclipse-plugin-1.7.0.jar.zip]
```

Comment:
I'm curious why is this old - is there a new plugin now that makes this old? Were you able to install the formatter? 

Side-note: i'm very tempted to tell everyone to always use Intellij IDE, it is also free and it's easier to configure. Intellij is also rapidly gaining market share, eclipse is somewhat, 'eclipsing'. A similar call out for the windows commands are probably needed on the intellij setup page as well.

Reply:
Ah, I see: 

> Current Google Java Format Plugin version works

Then please delete this line. Having 'old' documentation raises a lot more questions than it answers : +)